tls/s2n_resume.c

```diff
@@ -739,12 +739,11 @@ int s2n_encrypt_session_ticket(struct s2n_connection *conn, struct s2n_stuffer *
     return 0;
 }
 
-int s2n_decrypt_session_ticket(struct s2n_connection *conn)
+int s2n_decrypt_session_ticket(struct s2n_connection *conn, struct s2n_stuffer *from)
```

Comment:
Should `from` be just a `s2n_blob`? Or there is a value in moving the cursors outside of this function?

Reply:
It probably should be, but replacing it with a blob would require a more involved change to existing code, especially if I do it right and also make client_ticket_to_decrypt a blob too. I'd rather keep this change as minimal as possible.